src/actions/insert_actions.ts

```diff
@@ -248,6 +248,26 @@ export const insertLink: ActionSpec = {
   icon: "o-spreadsheet-Icon.INSERT_LINK",
 };
 
+export const insertCheckbox: ActionSpec = {
+  name: _t("Checkbox"),
+  execute: (env) => {
+    const { sheetId, col, row } = env.model.getters.getActivePosition();
+    const ranges = [env.model.getters.getRangeDataFromXc(sheetId, toXC(col, row))];
```

Comment:
should be the whole selected range, not only the anchor cell no ? At least it work like that in GSheet